### ISSUE TITLE
Disabled Backdrop or keyboard

### ISSUE THREAD

**Issue by reginaldogermano** [dixso/custombox#248]
How do you set the backdrop value to static 
or disabled keyboard

**Comment by dixso** [comment]
```js
// Instantiate new modal
new Custombox.modal({
  // Options
  content = {
    close: true
  }
});
```

**Comment by reginaldogermano** [comment]
Thank you.

 

De: Julio de la Calle [mailto:notifications@github.com] 
Enviada em: domingo, 8 de outubro de 2017 03:13
Para: dixso/custombox <custombox@noreply.github.com>
Cc: reginaldogermano <reginaldo.germano@gmail.com>; Author <author@noreply.github.com>
Assunto: Re: [dixso/custombox] Disabled Backdrop or keyboard (#248)

 

Closed #248 <https://github.com/dixso/custombox/issues/248> .
